src/main/resources/liveTemplates/ATG.xml

```diff
@@ -0,0 +1,15 @@
+<templateSet group="ATG">
```

Comment:
Split MR of unrelated requests on several described requests
PS. why you omit ATG AOP?
For us it's much more convenient just put @Transactional above method instead of supporting boiler-plate code